packages/libproxy.rb

```diff
@@ -2,36 +2,64 @@
 
 class Libproxy < Package
   description 'libproxy is a library that provides automatic proxy configuration management.'
-  homepage 'http://libproxy.github.io/libproxy/'
-  version '0.4.15'
+  homepage 'https://libproxy.github.io/libproxy/'
+  @_ver = '0.4.17'
+  version @_ver
   license 'LGPL-2.1+'
   compatibility 'all'
-  source_url 'https://github.com/libproxy/libproxy/releases/download/0.4.15/libproxy-0.4.15.tar.xz'
-  source_sha256 '654db464120c9534654590b6683c7fa3887b3dad0ca1c4cd412af24fbfca6d4f'
+  source_url 'https://github.com/libproxy/libproxy.git'
+  git_hashtag @_ver
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libproxy/0.4.15_armv7l/libproxy-0.4.15-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libproxy/0.4.15_armv7l/libproxy-0.4.15-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libproxy/0.4.15_i686/libproxy-0.4.15-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libproxy/0.4.15_x86_64/libproxy-0.4.15-chromeos-x86_64.tar.xz',
+  binary_url({
+    i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libproxy/0.4.17_i686/libproxy-0.4.17-chromeos-i686.tar.zst',
+  x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libproxy/0.4.17_x86_64/libproxy-0.4.17-chromeos-x86_64.tar.zst'
   })
-  binary_sha256 ({
-    aarch64: 'f0df6d6fbe49b9783ebe03bf922cb917714358dc4380c44dfa9b5bbdc747d831',
-     armv7l: 'f0df6d6fbe49b9783ebe03bf922cb917714358dc4380c44dfa9b5bbdc747d831',
-       i686: '00c146655e1bf4c437974ad59549b900924ac93f7de277477c196c56c44b7c73',
-     x86_64: '896073cbf37b750da1856e37c58111f4c1350196ddb6fef5ce7799197d7a9f27',
+  binary_sha256({
+    i686: '43aede3ba83cc73fb0cbe8040e8e0c36db419bee4932550eae31cf004351af07',
+  x86_64: '4b2a329496d51d6ef1c97bb30cf5530af784800c2df634abbae428f6cca4a04c'
   })
 
+  depends_on 'dbus'
+  depends_on 'vala' => :build
+  depends_on 'glib'
+
+  # ninja/samu doesn't work, makefiles do.
   def self.build
-    system './autogen.sh'
-    Dir.mkdir 'build'
-    Dir.chdir 'build' do
-      system "cmake .. -DLIB_INSTALL_DIR=#{CREW_LIB_PREFIX}"
+    Dir.mkdir 'builddir'
+    Dir.chdir 'builddir' do
+      # system "cmake -G Ninja #{CREW_CMAKE_OPTIONS} .."
+      system "cmake -G 'Unix Makefiles' #{CREW_CMAKE_OPTIONS} \
+              -DLIBEXEC_INSTALL_DIR=#{CREW_LIB_PREFIX} \
+              -DLIB_INSTALL_DIR=#{CREW_LIB_PREFIX} \
```

Comment:
This is wrong. `LIB_INSTALL_DIR` is either `LIB` or `LIB64`, I think.

You can remove this line and just use `CREW_CMAKE_LIBSUFFIX_OPTIONS`